specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/vmware.json

```diff
@@ -0,0 +1,1277 @@
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "Clusters"
+        ],
+        "summary": "Delete a cluster in a private cloud",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/clusterName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The delete has succeeded"
+          },
+          "202": {
+            "description": "Accepted. The delete will complete asynchronously"
+          },
+          "204": {
+            "description": "No Content. No cluster to delete"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Clusters_Delete": {
+            "$ref": "./examples/Clusters_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/listAdminCredentials": {
+      "post": {
+        "operationId": "PrivateClouds_ListAdminCredentials",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "List the admin credentials for the private cloud",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/AdminCredentials"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_ListAdminCredentials": {
+            "$ref": "./examples/PrivateClouds_ListAdminCredentials.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Quota": {
+      "description": "Subscription quotas",
+      "properties": {
+        "hostsRemaining": {
+          "description": "Remaining hosts quota by sku type",
+          "type": "object",
+          "readOnly": true,
+          "additionalProperties": {
+            "type": "integer",
+            "readOnly": true
+          }
```

Comment:
Additional properties make it more difficult to interact with the API/SDK because it's not clear what the keys/property names are actually allowed. Can we fully model the properties in the schema?

Reply:
I'm adding this to the list for the next API version.